changelogs/fragments/77755-pip-module-editable-flag-fix.yml

```diff
@@ -0,0 +1,4 @@
+bugfixes:
+  - pip - when editable set to true, '-e' flag is passed before each package.
+    When requirments file is used, editable flag does not pass '-e' to avoid
+    errors.
```

Comment:
```suggestion
  - pip - when editable is set to true, '-e' flag is passed before each package.
    When the requirements file is used, the editable flag does not pass '-e' to avoid
    errors.
```